assets/src/models/createDetourMachine.ts

```diff
@@ -188,6 +188,12 @@ export const createDetourMachine = setup({
                   },
                 },
                 Loading: {},
+                "No Route Error": {},
```

Comment:
Can we use punctuation in state names? This almost reads like there _isn't_ a route error, but it's like "No-Route Error", or "Error: No Route" would also work

Reply:
Great callout! Done! https://github.com/mbta/skate/compare/d35f2b2cd0934ed079f0bce75ff9b3d7d4b04ec2..ee0a7305856cca85e6b6ec0161ce34cf18b1a981